stages/org.osbuild.systemd

```diff
@@ -70,6 +77,13 @@ def main(tree, options):
     for service in masked_services:
         subprocess.run(["systemctl", "--root", tree, "mask", service], check=True)
 
+    if masked_generators:
+        generator_dir = f"{tree}/etc/systemd/system-generators"
```

Comment:
Given that the reset of the code is using `systemctl --root`, is there a specific reason to use something different here?

Reply:
I don't understand. systemctl doesn't support masking generators, only units. So, how would we use `systemctl --root`?

Reply:
I'm sorry, I was not aware of this. Maybe worth putting a small comment like: `systemctl does not support masking generators` ?